unifiedConfiguration.json

```diff
@@ -230,6 +230,14 @@
    "value" : ["AODSIM","MINIAODSIM","GEN-SIM-RAW","GEN-SIM-RECO","GEN-SIM-RECODEBUG","AOD","RECO","MINIAOD","ALCARECO","USER","RAW-RECO","RAWAODSIM","NANOAOD","NANOAODSIM","FEVT","PREMIX"],
    "description": "The data tiers that go to AnaOps"
  },
+ "tiers_to_rucio_relval": {
+   "value" : ["NANOAOD", "NANOAODSIM"],
+   "description": "The data tiers not to be considered for output data placement - RelVal workflows."
+ },
+ "tiers_to_rucio_nonrelval": {
+   "value" : [],
+   "description": "The data tiers not to be considered for output data placement - NonRelVal workflows."
```

Comment:
@todor-ivanov are you sure, you want to leave this empty? @amaltaro @nsmith- fyi

Reply:
yes, because we will start with the RelVal agent first. Then a few days later, we will switch all the other agents too.

Reply:
@amaltaro in this case the code related to this shouldn't be added to closor right now as an if to compare a tier with [] won't make sense.